libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -435,7 +435,7 @@ void AC_Avoid::adjust_velocity_polygon(float kP, float accel_cmss, Vector2f &des
 
     // for stopping
     float speed = safe_vel.length();
-    Vector2f stopping_point = position_xy + safe_vel*((2.0f + get_stopping_distance(kP, accel_cmss, speed))/speed);
+    Vector2f stopping_point_plus_margin = position_xy + safe_vel*((2.0f + margin_cm + get_stopping_distance(kP, accel_cmss, speed))/speed);
```

Comment:
`const`

Reply:
Also, what's up with the magic 2cm here?

Reply:
@peterbarker, good question about the 2cm.  I don't really think we need it now that we've added the margin.  At the same time, it does no harm..